.github/workflows/ci.yml

```diff
@@ -53,54 +55,60 @@ jobs:
         ghc-version: ${{ matrix.ghc }}
         cabal-version: ${{ matrix.cabal }}
 
-    - name: Configure
+    - if: matrix.ghc != '9.4.5' && matrix.ghc != '9.6.2'
+      name: Configure
```

Comment:
I am a kowainik assistant maintainer and I can merge this. However, could you please first explain why some of the steps have been disabled for 9.4.5 and 9.6.2?

Reply:
@tomjaguarpaw Not OP, but see this comment here: https://github.com/kowainik/summoner/pull/570#issuecomment-1673488453

Also see the summoner-cli.cabal: https://github.com/kowainik/summoner/blob/30b56041a501341dd0db41fe83a9cc161fc9552f/summoner-cli/summoner.cabal#L77

In essence, summoner-cli doesn't build with 9.4 and 9.6 due to too restrictive `base` (<4.17). It can still summon them, just doesn't build with them. 9.4 can probably be just bumped, but 9.6 has a compile issue: #575

Reply:
OK, let's fix summoner first to build with 9.4 and 9.6 rather than adding these special cases.

Reply:
Yeah, @Vekhir is right. At least at the time some of summoner's dependencies were not updated yet to those GHC versions.

@tomjaguarpaw do you want me to try building it again (and I could add some newer GHC versions while I'm at it)? Or do we want to merge this first and fix the other things later?

Reply:
I'll get summoner building on the latest GHCs, and when that's done you can rebase this MR on top (hopefully without special cases).

Reply:
OK, `main` now builds on up to 9.6. Could you please rebase your branch and remove the special cases?